.github/workflows/matrix.yaml

```diff
@@ -0,0 +1,54 @@
+name: "Telepresence Test Matrix"
+on:
+  # Using pull_request for ease of iteration, but "workflow_dispatch" will let us manually trigger with parameters.
+  pull_request: {}
+  workflow_dispatch:
+    inputs:
+      logLevel:
+        description: 'Log level'
+        required: true
+        default: 'warning'
+jobs:
+  telepresence_matrix:
+    strategy:
+      matrix:
+
+        # Comment out the full range of dimensions and use just a subset for faster iteration.
+
+        client_os: [ubuntu]
+        client_arch: [latest]
+#        client_os: [ubuntu-22.04, ubuntu-20.04]
+#        client_arch: [arm, x86]
+
+        client_telepresence_version: ["2.7"]
+#        client_telepresence_version: ["2.7", "2.6", "2.5"]
+
+        clusters:
+
+         - distribution: GKE
+           version: "1.23"
+#         - distribution: GKE
+#           version: "1.22"
+#         - distribution: GKE
+#           version: "1.21"
+
+         - distribution: AKS
+           version: "1.22"
+#         - distribution: AKS
+#           version: "1.21"
+#         - distribution: AKS
+#           version: "1.20"
+
+        cluster_telepresence_version:  ["none", "2.7"]
```

Comment:
What does `none` mean as a version? Is that the default?

Reply:
@aosoriodw That's from the example story, so I'm not sure, but my guess is that is intended to cover the case where there is no traffic manager installed in the cluster at all. Perhaps @bnfb could answer?